src/TypeScript/Sarif.SDK/package-lock.json

```diff
@@ -440,6 +470,7 @@
       "version": "0.4.12",
       "resolved": "https://registry.npmjs.org/memoizee/-/memoizee-0.4.12.tgz",
       "integrity": "sha512-sprBu6nwxBWBvBOh5v2jcsGqiGLlL2xr2dLub3vR8dnE8YB17omwtm/0NSHl8jjNbcsJd5GMWJAnTSVe/O0Wfg==",
+      "dev": true,
```

Comment:
did npm inject all these>?

Reply:
Yep, dependencies of various packages like TypeScript and the ts2json.

---
In reply to: [214737556](https://github.com/Microsoft/sarif-sdk/pull/1058#discussion_r214737556) [](ancestors = 214737556)